okhttp/src/main/java/okhttp3/internal/platform/AndroidPlatform.kt

```diff
@@ -194,22 +194,26 @@ class AndroidPlatform : Platform() {
     val isAndroid: Boolean = try {
       // Trigger an early exception over a fatal error, prefer a RuntimeException over Error.
       Class.forName("com.android.org.conscrypt.OpenSSLSocketImpl")
+
+      // account for android-all
+      check(Build.VERSION.SDK_INT > 0)
```

Comment:
Should fail before accidentally running in Android mode on JVM.  Defaults to 0 internally.